src/Compute/Compute/help/New-AzVM.md

```diff
@@ -390,7 +397,16 @@ Accept wildcard characters: False
 
 ### -Credential
 The administrator credentials for the VM.
-
+#### Username
```

Comment:
Microsoft docs doesn't support Lv4 header now. Please use another format.

Reply:
how about lv 5 and on?

Reply:
@wyunchi-ms